web/lumpTypeView.ts

```diff
@@ -471,3 +471,62 @@ function drawMapGeometry(
         }
     }
 }
+
+export function LumpTypeViewPlaypal(
+    options:{scaleX:number, scaleY:number} = {scaleX: 4, scaleY: 4}
+) {
+    return new LumpTypeView({
+        name: "Playpal",
+        icon: "assets/icons/lump-playpal.png",
+        view: (lump: WADLump, root: HTMLElement) => {
+            const container2 = util.createElement({class: "lump-view-playpal", appendTo: root});
+            const container = util.createElement({class: "lump-view-playpal-inner", appendTo: container2});
+            const numPals = Math.floor(lump.dataLength / 768); // Sometimes, the PLAYPAL has a comment at the end.
```

Comment:
Please use the `WADPalette.getPaletteCount` method instead

For example:

```suggestion
            const numPals = lumps.WADPalette.from(lump).getPaletteCount();
```

Reply:
How am I supposed to know that WADPalette is ready to use, though?

Reply:
I know it isn't obvious that it's there, but changing this code to use it reduces how much work needs to be done to maintain the code - if ever the specific way of figuring out the number of palettes should be changed, then using WADPalette this way means it only ever has to change in one place. It's also a bit clearer to read the code; "getPaletteCount" is more immediately clear in function than a bunch of math with magic constants.